grpc-interface-support/data/templates/cpp/ServiceNameServiceServerFactory.cc

```diff
@@ -38,7 +39,7 @@ std::unique_ptr<grpc::Server> ${{ service_name_camel_case }}ServiceServerFactory
     builder.RegisterService(service.get());
     // Finally assemble the server.
     std::unique_ptr<grpc::Server> server(builder.BuildAndStart());
-    std::cout << "Server listening on " << serviceLocation << std::endl;
+    velocitas::logger().info("Server listening on {}", serviceLocation);
```

Comment:
Should we add package_id(?) and service_name instead of just logging "Server" (which one?)? 
Not sure about if the replacement will work:
```suggestion
    velocitas::logger().info("Server ${{ package_id }}::${{ service_name }} listening on {}", serviceLocation);
```

Reply:
@BjoernAtBosch - refactored based on your suggestion. Works, see updated description in the PR description